### ISSUE TITLE
Gazebo Source Build Error

### ISSUE THREAD

**Issue by osrf-migration** [osrf/gazebo#2638]
**Original report ([archived issue](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/issues/2638)) by acxz (Bitbucket: [avcxz](https://bitbucket.org/%7B9dedb9bd-d101-4691-882b-c20ed6bbe205%7D/)).**

----------------------------------------
I am currently trying to build the latest release \(10.1\) from source. I have also installed the latest release sdformat and ignition packages from source.

‌

When trying to build gazebo dependencies \(sdformat and ignition packages\) from source, I follow this page \([http://gazebosim.org/tutorials?tut=install\_dependencies\_from\_source&cat=install](http://gazebosim.org/tutorials?tut=install_dependencies_from_source&cat=install)\) There is a versions section that lists the relevant sdformat/ignition versions to install for each Gazebo version. However, there is not an entry for the latest Gazebo release.

‌

Noting this I went ahead and just installed the latest releases for the sdformat/ignition packages dependencies. Going through the installation procedure I reach the following error:

```
-- BUILD ERRORS: These must be resolved before compiling.
-- 	Missing: SDF version >=6. Required for reading and writing SDF files.
-- 	Missing: Ignition msgs1 library (libignition-msgs-dev).
-- 	Missing: Ignition math (libignition-math4-dev)
-- 	Missing: Ignition Transport (libignition-transport4-dev)
-- END BUILD ERRORS
```

Okay so this tells me the information that should probably be on the versions page, I linked earlier. So I realize I need to downgrade my ignition-msgs, math, transport, but my sdformat package is 8.2 which is >=6. Should this be changed to `=6` instead?

TLDR:

Installation documentation for dependent packages of gazebo is outdated \([http://gazebosim.org/tutorials?tut=install\_dependencies\_from\_source&cat=install](http://gazebosim.org/tutorials?tut=install_dependencies_from_source&cat=install)\)

And, sdformat version checking is incorrectly listed in gazebo’s CMakeLists.txt or gazebo is not properly finding sdformat 8.2 even when it is supposedly works.
